image-bootstrap

```diff
@@ -1,11 +0,0 @@
-#! /usr/bin/env python2
-# Copyright (C) 2015 Sebastian Pipping <sebastian@pipping.org>
-# Licensed under AGPL v3 or later
-
-if __name__ == '__main__':
-    try:
-        from image_bootstrap.cli import main
-    except KeyboardInterrupt:
-        pass
-    else:
-        main()
```

Comment:
I would rather keep these two files around for now, on top of `entry_points`/`console_scripts`. If you don't see any problem with that, please revert removal. Thanks!